app/src/main/java/com/edwincobos/subscribersapp/commons/models/Subscriber.java

```diff
@@ -0,0 +1,402 @@
+package com.edwincobos.subscribersapp.commons.models;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.lang.reflect.Field;
+
+
+/**
+ * Subscriber class that represents each Github subscriber
+ * @author edwin.cobos
+ * @since 19/08/2016
+ */
+public class Subscriber {
+
+    @SerializedName("login")
```

Comment:
You do not need to define a serialized name if the attribute matches with the name